menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -321,12 +315,47 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
   def runCoordinator(coordPath: String, runtimeParams: RuntimeConfig): Future[String] = {
     getOozieClientWrap { oozieClient: OozieClient =>
       val conf = getOozieConf(oozieClient, runtimeParams)
-      conf.setProperty(OozieClient.COORDINATOR_APP_PATH, s"$coordPath");
-      // submit and start the workflow job
+      conf.setProperty(OozieClient.COORDINATOR_APP_PATH, s"$coordPath")
       oozieClient.submit(conf)
     }
   }
 
+  /**
+   * Run a workflow now
+   */
+  def runWorkflow(wfPath: String, runtimeParams: RuntimeConfig, reportDate: String): Future[String] = {
+    getOozieClient { oozieClient: OozieClient =>
```

Comment:
```suggestion
    getOozieClient { oozieClient =>
```
Why do all of these have an explicit type?